src/audio/pipewire.c

```diff
@@ -113,6 +113,9 @@ static void on_process(void *userdata)
     if (b->requested > 0)
     {
         total_number_of_bytes_in_pipewire_buffer = SPA_MIN(total_number_of_bytes_in_pipewire_buffer, b->requested * state->stride); // because this value is given in frames/samples, and we want bytes
+        // if there's anything else to fill and the ring buffer doesn't contain enough at this point, fill the difference with silence
+        must_load_with_silence = total_number_of_bytes_in_pipewire_buffer - available_for_loading;
```

Comment:
No :) I meant to move it in the other `if (b->requested)` block, where it is actually used. It's no use computing it that early, better compute it just before using it.

Reply:
ahh ok, will do, seemned weird to me that that's what you want, I get it now

Reply:
done now in the latest commit